### PR TITLE
Doc: To make clear not a new operator; clearer?

### PR DESCRIPTION
**Pull request by PallHaraldsson**
I thought one more operator (that I didn't know..)
[Skip ci]

### CODE REVIEW

**Comment by ararslan** [comment]
Skipping CI is done via `[ci skip]`. `[Skip ci]` as you've done doesn't trigger the skip, so it's still running.

File: NEWS.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Language changes
     module. For instance, `Base.r"x"` is now parsed as `Base.@r_str "x"`.
     Previously, this syntax parsed as an implicit multiplication. ([#18690])
 
-  * For every binary operator `⨳`, `a .⨳ b` is now automatically equivalent to
+  * For every binary operator `⨳` (`⨳` is only a placeholder; isn't a new operator), `a .⨳ b` is now automatically equivalent to
```
Comment (ararslan) [inline NEWS.md]: I'd probably simplify this to "(where `⨳` is a placeholder)"

Comment (stevengj) [inline NEWS.md]: Actually, `⨳` is an example of a real binary operator (which doesn't have a default definition).

Maybe
```
For every binary operator, such as `⨳`, .....
```

Comment (stevengj) [inline NEWS.md]: Or better yet:
```
For every binary operator, even user-defined operators such as `⨳`, ...
```

**Comment by ViralBShah** [comment]
I realize this is for 0.6 release notes, and hence am closing it.

**Pull request closed without merging**